app/bundles/FormBundle/EventListener/DetermineWinnerSubscriber.php

```diff
@@ -57,7 +59,8 @@ public static function getSubscribedEvents()
      */
     public function onDetermineSubmissionWinner(DetermineWinnerEvent $event)
     {
-        $repo       = $this->em->getRepository('MauticFormBundle:Submission');
+        /** @var SubmissionRepository $repo */
+        $repo       = $this->em->getRepository(Submission::class);
```

Comment:
Can we directly inject the `SubmissionRepository` here instead of the entity manager to get the repo?

Reply:
https://github.com/mautic/mautic/pull/8148/commits/499dd9ce1c103139d523f3e8638456d9fd2c35b6